arceos-vmm/scripts/make/qemu.mk

```diff
@@ -24,7 +24,7 @@ qemu_args-aarch64 := \
   -machine virt \
   -kernel $(OUT_BIN)
 
-qemu_args-y := -m 2G -smp $(SMP) $(qemu_args-$(ARCH))
+qemu_args-y := -m 128M -smp $(SMP) $(qemu_args-$(ARCH))
```

Comment:
The memory mapping was wrong in aarch64 before. To avoid the bug, we scaled the QEMU memory to 2GB, even though the actual runtime memory requirement was much less. Now that we've fixed the mapping bug, we've reverted the QEMU memory setting to its original value.

Reply:
I think 2G is ok.